app/services/sufia/admin_set_service.rb

```diff
@@ -1,5 +1,14 @@
 module Sufia
   # Returns AdminSets that the current user has permission to use.
   class AdminSetService < CurationConcerns::AdminSetService
+    # @param [Symbol] access :read or :edit
+    def select_options(access = :read)
+      search_results(access).map do |element|
+        permission_template = PermissionTemplate.find_by_admin_set_id(element.id)
+        visibility = permission_template.visibility if permission_template
```

Comment:
When you get to the specs, we should make sure to test the case where `permission_template` is `nil`.